indra/llcommon/llsys.cpp

```diff
@@ -214,7 +214,7 @@ LLOSInfo::LLOSInfo() :
 		DWORD cbData(sizeof(DWORD));
 		DWORD data(0);
 		HKEY key;
-		BOOL ret_code = RegOpenKeyExW(HKEY_LOCAL_MACHINE, TEXT("SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion"), 0, KEY_READ, &key);
+        bool ret_code = RegOpenKeyExW(HKEY_LOCAL_MACHINE, TEXT("SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion"), 0, KEY_READ, &key);
```

Comment:
this should stay as BOOL if it is being compared against WinAPI `ERROR_SUCCESS`

Reply:
It should not stay BOOL, but get changed to LSTATUS

Reply:
Changed it to LSTATUS.